JapaneseFilter/sections/specific.txt

```diff
@@ -3036,6 +3036,10 @@ nc-net.or.jp##.box-pr-image-01
 mori.nc-net.or.jp###ok_ad_side
 gyutte.jp###gt_sp_all_header
 gyutte.jp###gt_sp_all_footer
+japanjournals.com###easycookieinfo
```

Comment:
Please move it to Cookie Notices filter, it can be added to current rule: https://github.com/AdguardTeam/AdguardFilters/blob/ff067d646e6ff9f9ba2451745dd187f05a30c29f/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L6974